doc/mgr/orchestrator.rst

```diff
@@ -273,13 +273,85 @@ Placement Specification
 
 In order to allow the orchestrator to deploy a *service*, it needs to
 know how many and where it should deploy *daemons*. The orchestrator 
-defines a placement specification:
+defines a placement specification that can either be passed as command line arguments.
```

Comment:
Is this meant to be "can either be passed as .... or...." ? are you missing a second half here? If not, see my suggestion below